conventions/documentation/quick-starts.md

```diff
@@ -0,0 +1,205 @@
+  * **Team responsible:** Product Management; UXD Design; possibly UXD Research
+  * **Time frame:** Before epic prioritization
+  * **Definition of done:** Product Management, UXD Design, UXD Research, and any other groups in this step agree on the product area to prioritize for a quick start.
+2. Plan user flows and interaction design. (Optional)
+  * UXD Designer, UXD Content Strategist, and CCS Content Strategist and/or Writer meet with Product Manager to determine the quick start’s goal and where we need to reduce friction or confusion in completing some tasks.
+  * Collaborate on what the ideal user flow would look like. For example, what are the tasks and the sub tasks that users need to do in order to complete the quick start?
+  * Work with Abi to figure out who we work with in CCS and tag in JIRA story.
+    * Consider prioritization (track in JIRA); use template.
+    * Maybe need a separate process for external teams; have UXD Content & CCS just do a quick run-through.
+  * At this point, the UXD Designer is putting designs into Sketch and Marvel and getting feedback on interactions (not content yet). These will not be hi-fi designs.
+  * **Team responsible:** UXD Design (including Dev Tools UXD); Product Management; UXD Content Strategy; CCS
+  * **Time frame:** Conceptual design story (Sprint 1)
+  * **Definition of done:**
+    * UXD Designer creates interaction designs (not hi-fi) in Sketch and Marvel.
+    * Product Manager reviews the interactions and provides feedback.
+    * Stakeholder review (by Product Management and Dev Lead) is complete and all feedback is applied.
+
+#### Phase 2: First draft
+UXD Designer writes the first draft of the quick start, using the following as references:
+* Interaction design plans (to align with user flow)
+* Technical documentation (to avoid contradiction and duplication of effort)
+* Red Hat brand standards/PatternFly UX writing style guide (to align with brand voice and UX writing best practices)
+* CCS style resources (to align with docs best practices)
+* Quick start writing template (to maintain consistency)
+* **Team responsible:** UXD Design
+* **Time frame:** After conceptual design story (Sprint 2)
+* **Definition of done:**
+  * All steps, descriptions, messaging, and other content is completely written.
+  * Draft is ready for CCS review.
+
+#### Phase 3: CCS review
+CCS Technical Writer conducts a review of the draft.
+
+During this step, have a checkpoint/feedback loop with CCS to give UX feedback on existing documentation. A smaller group can work together and identify what it will take to make existing documentation consumable for quick starts.
+
+* **Team responsible:** CCS Technical Writing; UXD Design
+* **Time frame:** Sprint 2
+* **Definition of done:**
+  * CCS Technical Writer reviewed all content and left edits/comments.
+  * UXD Designer addressed all feedback.
+
+#### Phase 4: UXD Content review
+Abi conducts a content review of the draft.
```

Comment:
Is this more work than just one person can manage?

Reply:
Chances are, Alana and I will share this responsibility. 

If bandwidth is an issue for any of the content review stakeholders (UXD or CCS), we can always adjust reviewals for each quick start as needed. I think we should at least aim to get one content review in per quick start. So if CCS doesn't have bandwidth for a particular quick start review, we still have UXD Content. If UXD Content doesn't have bandwidth for a particular quick start review, we still have CCS. Ideally, we'll have both—but we'll need to try out the process and be flexible depending on what everyone's bandwidth looks like. 👍